### PR TITLE
Using lower case microsoft in package names

### PR DESCRIPTION
**Pull request by arschles**
Since the GitHub Microsoft organization name has been changed to microsoft, package names and links in docs files have been downcased

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,109 @@
+module github.com/arschles/presidio
```
Comment (omri374) [inline go.mod]: Why is this file added?

**Comment by omri374** [comment]
Thanks @arschles. There are (at least) two new files: go.mod and go.sum. What's the purpose of these files?
In addition, the Microsoft name was lower cased even in documentation and other non-code places. 
Would you prefer to fix this PR or should we close this one and start a new one?

File: presidio-analyzer/tests/data/demo.txt [hunk 2]
```diff
@@ -22,5 +22,5 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all r
 epos using our CLA.
 
-This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact 
-opencode@microsoft.com with any additional questions or comments.
\ No newline at end of file
+This project has adopted the microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact
```
Comment (omri374) [inline presidio-analyzer/tests/data/demo.txt]:
```suggestion
This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact 
```

File: presidio-anonymizer/cmd/presidio-anonymizer/anonymizer/anonymizer_test.go [hunk 3]
```diff
@@ -410,7 +410,7 @@ This project welcomes contributions and suggestions. Most contributions require
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the 
 PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact 
+This project has adopted the microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact
```
Comment (omri374) [inline presidio-anonymizer/cmd/presidio-anonymizer/anonymizer/anonymizer_test.go]:
```suggestion
This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact 
```

File: presidio-anonymizer/cmd/presidio-anonymizer/anonymizer/anonymizer_test.go [hunk 4]
```diff
@@ -437,7 +437,7 @@ This project welcomes contributions and suggestions. Most contributions require
...
```
Comment (omri374) [inline presidio-anonymizer/cmd/presidio-anonymizer/anonymizer/anonymizer_test.go]:
```suggestion
This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact 
```

File: presidio-tester/cmd/presidio-tester/testdata/ocr-result.json [hunk 5]
```diff
@@ -1 +1 @@
...
...sition":781,"endPosition":791},{"xLocation":833,"width":1000,"yLocation":914,"height":950,"text":"questions","startPosition":792,"endPosition":801},{"xLocation":1013,"width":1047,"yLocation":921,"height":943,"text":"or","startPosition":802,"endPosition":804},{"xLocation":27,"width":219,"yLocation":971,"height":998,"text":"comments.","startPosition":805,"endPosition":814}],"text":"This project is created by David Johnson and welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com or contact (212) 555-1234. When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA| This project has adopted the microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments. "}
```
Comment (omri374) [inline presidio-tester/cmd/presidio-tester/testdata/ocr-result.json]:
```suggestion
{"Boundingboxes":[{"xLocation":27,"width":99,"yLocation":20,"height":51,"text":"This","endPosition":4},{"xLocation":112,"width":229,"yLocation":22,"height":59,"text":"project","startPosition":5,"endPosition":12},{"xLocation":242,"width":267,"yLocation":22,"height":51,"text":"is","startPosition":13,"endPosition":15},{"xLocation":280,"width":407,"yLocation":20,"height":51,"text":"created","startPosition":16,"endPosition":23},{"xLocation":421,"width":459,"yLocation":20,"height":59,"text":"by","startPosition":24,"endPosition":26},{"xLocation":472,"width":563,"yLocation":20,"height":51,"text":"David","startPosition":27,"endPosition":32},{"xLocation":576,"width":723,"yLocation":20,"height":51,"text":"Johnson","startPosition":33,"endPosition":40},{"xLocation":737,"width":798,"yLocation":20,"height":51,"text":"and","startPosition":41,"endPosition":44},{"xLocation":811,"width":985,"yLocation":20,"height":51,"text":"welcomes","startPosition":45,"endPosition":53},{"xLocation":27,"width":253,"yLocation":76,"height":107,"text":"contributions","startPosition":54,"endPosition":67},{"xLocation":267,"width":327,"yLocation":76,"height":107,"text":"and","startPosition":68,"endPosition":71},{"xLocation":341,"width":559,"yLocation":78,"height":114,"text":"suggestions.","startPosition":72,"endPosition":84},{"xLocation":574,"width":659,"yLocation":78,"height":107,"text":"Most","startPosition":85,"endPosition":89},{"xLocation":671,"width":897,"yLocation":76,"height":107,"text":"contributions","startPosition":90,"endPosition":103},{"xLocation":911,"width":1028,"yLocation":78,"height":114,"text":"require","startPosition":104,"endPosition":111},{"xLocation":26,"width":86,"yLocation":141,"height":171,"text":"you","startPosition":112,"endPosition":115},{"xLocation":97,"width":131,"yLocation":136,"height":163,"text":"to","startPosition":116,"endPosition":118},{"xLocation":144,"width":238,"yLocation":141,"height":170,"text":"agree","startPosition":119,"endPosition":124},{"xLocation":249,"width":282,"yLocation":136,"height":163,"text":"to","startPosition":125,"endPosition":127},{"xLocation":295,"width":312,"yLocation":141,"height":163,"text":"a","startPosition":128,"endPosition":129},{"xLocation":326,"width":519,"yLocation":132,"height":163,"text":"Contributor","startPosition":130,"endPosition":141},{"xLocation":532,"width":662,"yLocation":134,"height":163,"text":"License","startPosition":142,"endPosition":149},{"xLocation":673,"width":862,"yLocation":134,"height":170,"text":"Agreement","startPosition":150,"endPosition":159},{"xLocation":875,"width":970,"yLocation":131,"height":171,"text":"(CLA)","startPosition":160,"endPosition":165},{"xLocation":28,"width":183,"yLocation":188,"height":226,"text":"declaring","startPosition":166,"endPosition":175},{"xLocation":195,"width":262,"yLocation":188,"height":219,"text":"that","startPosition":176,"endPosition":180},{"xLocation":274,"width":333,"yLocation":197,"height":227,"text":"you","startPosition":181,"endPosition":184},{"xLocation":347,"width":405,"yLocation":188,"height":219,"text":"have","startPosition":185,"endPosition":189},{"xLocation":408,"width":490,"yLocation":188,"height":219,"text":"the","startPosition":190,"endPosition":193},{"xLocation":503,"width":578,"yLocation":188,"height":226,"text":"right","startPosition":194,"endPosition":199},{"xLocation":589,"width":630,"yLocation":192,"height":224,"text":"to,","startPosition":200,"endPosition":203},{"xLocation":643,"width":703,"yLocation":188,"height":219,"text":"and","startPosition":204,"endPosition":207},{"xLocation":717,"width":849,"yLocation":188,"height":227,"text":"actually","startPosition":208,"endPosition":216},{"xLocation":861,"width":910,"yLocation":188,"height":224,"text":"do,","startPosition":217,"endPosition":220},{"xLocation":923,"width":1010,"yLocation":192,"height":226,"text":"grant","startPosition":221,"endPosition":226},{"xLocation":28,"width":66,"yLocation":252,"height":274,"text":"us","startPosition":227,"endPosition":229},{"xLocation":77,"width":131,"yLocation":243,"height":274,"text":"the","startPosition":230,"endPosition":233},{"xLocation":144,"width":239,"yLocation":243,"height":281,"text":"rights","startPosition":234,"endPosition":240},{"xLocation":250,"width":283,"yLocation":247,"height":274,"text":"to","startPosition":241,"endPosition":243},{"xLocation":297,"width":356,"yLocation":252,"height":274,"text":"use","startPosition":244,"endPosition":247},{"xLocation":367,"width":441,"yLocation":252,"height":282,"text":"your","startPosition":248,"endPosition":252},{"xLocation":453,"width":668,"yLocation":243,"height":274,"text":"contribution.","startPosition":253,"endPosition":266},{"xLocation":683,"width":736,"yLocation":245,"height":274,"text":"For","startPosition":267,"endPosition":270},{"xLocation":748,"width":868,"yLocation":243,"height":279,"text":"details,","startPosition":271,"endPosition":279},{"xLocation":880,"width":949,"yLocation":245,"height":274,"text":"visit","startPosition":280,"endPosition":285},{"xLocation":28,"width":466,"yLocation":299,"height":337,"text":"https://cla.microsoft.com","startPosition":286,"endPosition":311},{"xLocation":479,"width":513,"yLocation":308,"height":330,"text":"or","startPosition":312,"endPosition":314},{"xLocation":525,"width":653,"yLocation":303,"height":330,"text":"contact","startPosition":315,"endPosition":322},{"xLocation":666,"width":754,"yLocation":298,"height":338,"text":"(212)","startPosition":323,"endPosition":328},{"xLocation":769,"width":938,"yLocation":301,"height":330,"text":"555-1234.","startPosition":329,"endPosition":338},{"xLocation":27,"width":102,"yLocation":355,"height":386,"text":"When","startPosition":339,"endPosition":343},{"xLocation":105,"width":193,"yLocation":364,"height":394,"text":"you","startPosition":344,"endPosition":347},{"xLocation":207,"width":324,"yLocation":355,"height":386,"text":"submit","startPosition":348,"endPosition":354},{"xLocation":337,"width":354,"yLocation":364,"height":386,"text":"a","startPosition":355,"endPosition":356},{"xLocation":368,"width":426,"yLocation":355,"height":393,"text":"pull","startPosition":357,"endPosition":361},{"xLocation":440,"width":575,"yLocation":359,"height":393,"text":"request,","startPosition":362,"endPosition":370},{"xLocation":588,"width":605,"yLocation":364,"height":386,"text":"a","startPosition":371,"endPosition":372},{"xLocation":618,"width":742,"yLocation":355,"height":386,"text":"CLA-bot","startPosition":373,"endPosition":380},{"xLocation":744,"width":821,"yLocation":355,"height":386,"text":"will","startPosition":381,"endPosition":385},{"xLocation":28,"width":259,"yLocation":410,"height":449,"text":"automatically","startPosition":386,"endPosition":399},{"xLocation":271,"width":444,"yLocation":410,"height":441,"text":"determine","startPosition":400,"endPosition":409},{"xLocation":455,"width":593,"yLocation":410,"height":441,"text":"whether","startPosition":410,"endPosition":417},{"xLocation":604,"width":663,"yLocation":419,"height":449,"text":"you","startPosition":418,"endPosition":421},{"xLocation":677,"width":757,"yLocation":410,"height":441,"text":"need","startPosition":422,"endPosition":426},{"xLocation":769,"width":802,"yLocation":414,"height":441,"text":"to","startPosition":427,"endPosition":429},{"xLocation":816,"width":940,"yLocation":410,"height":448,"text":"provide","startPosition":430,"endPosition":437},{"xLocation":953,"width":970,"yLocation":419,"height":441,"text":"a","startPosition":438,"endPosition":439},{"xLocation":984,"width":1054,"yLocation":412,"height":441,"text":"CLA","startPosition":440,"endPosition":443},{"xLocation":28,"width":88,"yLocation":466,"height":497,"text":"and","startPosition":444,"endPosition":447},{"xLocation":102,"width":252,"yLocation":466,"height":497,"text":"decorate","startPosition":448,"endPosition":456},{"xLocation":262,"width":316,"yLocation":466,"height":497,"text":"the","startPosition":457,"endPosition":460},{"xLocation":29,"width":74,"yLocation":524,"height":552,"text":"PR","startPosition":461,"endPosition":463},{"xLocation":86,"width":311,"yLocation":522,"height":561,"text":"appropriately","startPosition":464,"endPosition":477},{"xLocation":323,"width":404,"yLocation":521,"height":561,"text":"(e.g.,","startPosition":478,"endPosition":484},{"xLocation":417,"width":503,"yLocation":522,"height":558,"text":"label,","startPosition":485,"endPosition":491},{"xLocation":516,"width":702,"yLocation":521,"height":561,"text":"comment).","startPosition":492,"endPosition":501},{"xLocation":715,"width":830,"yLocation":522,"height":561,"text":"Simply","startPosition":502,"endPosition":508},{"xLocation":841,"width":946,"yLocation":522,"height":553,"text":"follow","startPosition":509,"endPosition":515},{"xLocation":956,"width":1010,"yLocation":522,"height":553,"text":"the","startPosition":516,"endPosition":519},{"xLocation":29,"width":229,"yLocation":579,"height":608,"text":"instructions","startPosition":520,"endPosition":532},{"xLocation":243,"width":388,"yLocation":577,"height":615,"text":"provided","startPosition":533,"endPosition":541},{"xLocation":402,"width":439,"yLocation":577,"height":616,"text":"by","startPosition":542,"endPosition":544},{"xLocation":450,"width":504,"yLocation":577,"height":608,"text":"the","startPosition":545,"endPosition":548},{"xLocation":517,"width":578,"yLocation":577,"height":608,"text":"bot.","startPosition":549,"endPosition":553},{"xLocation":591,"width":655,"yLocation":579,"height":608,"text":"You","startPosition":554,"endPosition":557},{"xLocation":667,"width":722,"yLocation":577,"height":607,"text":"will","startPosition":558,"endPosition":562},{"xLocation":735,"width":805,"yLocation":577,"height":616,"text":"only","startPosition":563,"endPosition":567},{"xLocation":818,"width":897,"yLocation":577,"height":608,"text":"need","startPosition":568,"endPosition":572},{"xLocation":910,"width":943,"yLocation":581,"height":608,"text":"to","startPosition":573,"endPosition":575},{"xLocation":956,"width":997,"yLocation":577,"height":608,"text":"do","startPosition":576,"endPosition":578},{"xLocation":26,"width":88,"yLocation":633,"height":664,"text":"this","startPosition":579,"endPosition":583},{"xLocation":101,"width":183,"yLocation":642,"height":664,"text":"once","startPosition":584,"endPosition":588},{"xLocation":196,"width":289,"yLocation":642,"height":664,"text":"across","startPosition":589,"endPosition":595},{"xLocation":292,"width":358,"yLocation":633,"height":664,"text":"all","startPosition":596,"endPosition":599},{"xLocation":373,"width":466,"yLocation":642,"height":671,"text":"repos","startPosition":600,"endPosition":605},{"xLocation":480,"width":570,"yLocation":635,"height":671,"text":"using","startPosition":606,"endPosition":611},{"xLocation":583,"width":638,"yLocation":642,"height":664,"text":"our","startPosition":612,"endPosition":615},{"xLocation":650,"width":734,"yLocation":626,"height":673,"text":"CLA|","startPosition":616,"endPosition":620},{"xLocation":27,"width":99,"yLocation":745,"height":776,"text":"This","startPosition":621,"endPosition":625},{"xLocation":112,"width":229,"yLocation":747,"height":784,"text":"project","startPosition":626,"endPosition":633},{"xLocation":242,"width":301,"yLocation":745,"height":776,"text":"has","startPosition":634,"endPosition":637},{"xLocation":314,"width":452,"yLocation":745,"height":783,"text":"adopted","startPosition":638,"endPosition":645},{"xLocation":464,"width":518,"yLocation":745,"height":776,"text":"the","startPosition":646,"endPosition":649},{"xLocation":532,"width":696,"yLocation":745,"height":776,"text":"microsoft","startPosition":650,"endPosition":659},{"xLocation":708,"width":795,"yLocation":747,"height":783,"text":"Open","startPosition":660,"endPosition":664},{"xLocation":808,"width":927,"yLocation":747,"height":776,"text":"Source","startPosition":665,"endPosition":671},{"xLocation":939,"width":1027,"yLocation":745,"height":776,"text":"Code","startPosition":672,"endPosition":676},{"xLocation":27,"width":62,"yLocation":800,"height":831,"text":"of","startPosition":677,"endPosition":679},{"xLocation":73,"width":225,"yLocation":800,"height":831,"text":"Conduct.","startPosition":680,"endPosition":688},{"xLocation":240,"width":293,"yLocation":802,"height":831,"text":"For","startPosition":689,"endPosition":692},{"xLocation":306,"width":392,"yLocation":809,"height":831,"text":"more","startPosition":693,"endPosition":697},{"xLocation":406,"width":602,"yLocation":800,"height":831,"text":"information","startPosition":698,"endPosition":709},{"xLocation":616,"width":674,"yLocation":809,"height":831,"text":"see","startPosition":710,"endPosition":713},{"xLocation":685,"width":739,"yLocation":800,"height":831,"text":"the","startPosition":714,"endPosition":717},{"xLocation":751,"width":839,"yLocation":800,"height":831,"text":"Code","startPosition":718,"endPosition":722},{"xLocation":851,"width":886,"yLocation":800,"height":831,"text":"of","startPosition":723,"endPosition":725},{"xLocation":897,"width":1040,"yLocation":800,"height":831,"text":"Conduct","startPosition":726,"endPosition":733},{"xLocation":29,"width":94,"yLocation":858,"height":891,"text":"FAQ","startPosition":734,"endPosition":737},{"xLocation":107,"width":141,"yLocation":865,"height":887,"text":"or","startPosition":738,"endPosition":740},{"xLocation":152,"width":281,"yLocation":860,"height":887,"text":"contact","startPosition":741,"endPosition":748},{"xLocation":27,"width":485,"yLocation":912,"height":951,"text":"opencode@microsoft.com","startPosition":749,"endPosition":771},{"xLocation":498,"width":568,"yLocation":912,"height":943,"text":"with","startPosition":772,"endPosition":776},{"xLocation":582,"width":640,"yLocation":921,"height":951,"text":"any","startPosition":777,"endPosition":780},{"xLocation":652,"width":819,"yLocation":912,"height":943,"text":"additional","startPosition":781,"endPosition":791},{"xLocation":833,"width":1000,"yLocation":914,"height":950,"text":"questions","startPosition":792,"endPosition":801},{"xLocation":1013,"width":1047,"yLocation":921,"height":943,"text":"or","startPosition":802,"endPosition":804},{"xLocation":27,"width":219,"yLocation":971,"height":998,"text":"comments.","startPosition":805,"endPosition":814}],"text":"This project is created by David Johnson and welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com or contact (212) 555-1234. When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA| This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments. "}
```

**Comment by omri374** [comment]
Opened under #157

**Comment by arschles** [comment]
Thanks for your comments @omri374! Sorry for the super long delay. The `go.mod` file is for [Go modules](https://blog.golang.org/using-go-modules)
